traceroute/help.md

```diff
@@ -69,15 +69,15 @@ Example output:
 
 ### Triggers
 
-This plugin does not contain any triggers.
+_This plugin does not contain any triggers._
 
 ### Custom Output Types
 
 _This plugin does not contain any custom output types._
 
 ## Troubleshooting
 
-This plugin does not contain any troubleshooting information.
+_This plugin does not contain any troubleshooting information._
```

Comment:
I saw you manually fixing these so I put a mass fix PR.

https://github.com/rapid7/insightconnect-plugins/pull/215
https://github.com/rapid7/insightconnect-plugins/pull/215